Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -392,7 +392,7 @@ script:
       - 'false'
     - description: Username to assign the detections to. (This is usually the user’s email address, but may vary based on your configuration). username and assigned_to_uuid are mutually exclusive.
       name: username
-    description: Resolves and updates a detection using the provided arguments. At least one optional argument must be passed, otherwise no change will take place.
+    description: Resolves and updates a detection using the provided arguments. At least one optional argument must be passed, otherwise no change will take place, does not support IDP detections.
```

Comment:
```suggestion
    description: Resolves and updates a detection using the provided arguments. At least one optional argument must be passed, otherwise no change will take place. Note that it does not support IDP detections.
```
Also in README